irc_im.c

```diff
@@ -969,6 +969,31 @@ static gboolean bee_irc_channel_chat_invite( irc_channel_t *ic, irc_user_t *iu )
 	return TRUE;
 }
 
+static void bee_irc_channel_chat_kick( irc_channel_t *ic, irc_user_t *iu, const char *msg )
+{
+	struct groupchat *c = ic->data;
+	bee_user_t *bu = iu->bu;
+	
+	if( bu == NULL )
+		return;
+	
+	if( c && ( iu->bu->ic != c->ic ) )
+	{
+		irc_send_num( ic->irc, 482, "%s :Can't mix different IM networks in one groupchat", ic->name );
+		return;
+	}
+	else if( !c->ic->acc->prpl->chat_kick )
+	{
+		irc_send_num( ic->irc, 482, "%s :IM protocol does not support room kicking", ic->name );
+		return;
+	}
+	
+	/* TODO: Need more const goodness here, sigh */
```

Comment:
I was only doing what was [already](https://github.com/bitlbee/bitlbee/blob/1fa510944f3f95f1907422dc8f7cfe9238b99cd0/irc_im.c#L930]) done.  I guess the other functions are non-constant for Gaim compatibility?  In that case, I guess it does not hurt to be "inconsistent" for the sake of sanity?